src/components/FlagSelect.tsx

```diff
@@ -1,3 +1,89 @@
-export function Welcome() {
-    return <h1>something</h1>;
-  }
\ No newline at end of file
+import React, { useEffect, useState } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import Dialog from "@material-ui/core/Dialog";
+import Australia from "/home/ishfaq/training-app/src/flags/Australia.png"
+import america from "/home/ishfaq/training-app/src/flags/america.png"
+import { Typography } from "@material-ui/core";
+
+const flags = {
```

Comment:
you could make this like this
const flags=[
{
img: "america",
code: "+1-United States"
},
{
img: "Australia",
code: "+61-Australia"
}
]

Reply:
it makes more sense